Frontend/src/pages/home/index.jsx

```diff
@@ -1,53 +1,141 @@
-          available, so don't miss this chance to be a part of the most
-          anticipated Rubik's Cube competition of the year. Whether you're here
-          to compete or simply soak up the excitement,{' '}
-          {competitionInfo.short_name} promises an unforgettable experience for
-          everyone.
-        </span>
+      <div className={styles.requirements}>
+        <div>Registration Requirements:</div>
+        <div>[INSERT ORGANIZER MESSAGE REGARDING REQUIREMENTS]</div>
+        <div>
+          <Button>View All</Button>
+        </div>
       </div>
+      {competitionInfo.information ? (
+        <div className={styles.information}>
+          <div className={styles.informationHeader}>Information:</div>
+          <div
+            className={styles.informationText}
+            dangerouslySetInnerHTML={{
+              __html: marked(competitionInfo.information),
+            }}
+          />
+        </div>
+      ) : (
+        ''
+      )}
       <div className={styles.registrationPeriod}>
-        <h2>Registration Period:</h2>
-        Online registration opened{' '}
-        {new Date(competitionInfo.registration_open).toString()} and will close{' '}
-        {new Date(competitionInfo.registration_close).toString()}.
+        <div className={styles.registrationHeader}>Registration Period:</div>
+        <div className={styles.registrationPeriodText}>
+          {new Date(competitionInfo.registration_open) < new Date()
+            ? `Registration opened ${moment(
+                competitionInfo.registration_open
+              ).calendar()} and will close ${moment(
+                competitionInfo.registration_close
+              ).format('ll')}`
+            : `Registration will open ${moment(
+                competitionInfo.registration_open
+              ).calendar()}`}
+        </div>
+      </div>
+      <div className={styles.details}>
+        <div>
+          <span className={styles.detailHeader}>Date: </span>
+          {moment(competitionInfo.start_date).format('ll')}{' '}
+        </div>
+        <div>
+          <span className={styles.detailHeader}>Start Time: </span>
+          XX:XX PM
+        </div>
+        <div>
+          <span className={styles.detailHeader}>City: </span>
+          {competitionInfo.city}, {competitionInfo.country_iso2}
+        </div>
+        <div>
+          <span className={styles.detailHeader}>Venue: </span>
+          <span
+            dangerouslySetInnerHTML={{ __html: marked(competitionInfo.venue) }}
+          />
+          <ul>
+            <li>
+              <span className={styles.detailHeader}>Address: </span>
+              {competitionInfo.venue_address}
+            </li>
+            {competitionInfo.venue_details ? (
+              <li>
+                <span className={styles.detailHeader}>Details: </span>
+                {competitionInfo.venue_details}
+              </li>
+            ) : (
+              ''
+            )}
```

Comment:
There are a few instances of `a ? b : ''` which can be replaced with `a && b`, unless the empty string is important for something.

Reply:
true, I keep on forgetting that...